prov/efa/src/rxr/rxr_ep.c

```diff
@@ -1093,8 +1093,10 @@ int rxr_ep_init(struct rxr_ep *ep)
 
 	if (rxr_env.rx_copy_ooo) {
 		ret = ofi_bufpool_create(&ep->rx_ooo_pkt_pool, entry_sz,
-					 RXR_BUF_POOL_ALIGNMENT, 0,
-					 rxr_env.recvwin_size, 0);
+					 RXR_BUF_POOL_ALIGNMENT,
+					 rxr_env.recvwin_size,
```

Comment:
setting max_cnt as rxr_env.recvwin_size seems arbitrary. Just leave it as 0?

Reply:
fixed in new revision.